server_admin/topics/export-import.adoc

```diff
@@ -85,6 +85,11 @@ Examples:
 * -Dkeycloak.import=/tmp/realm1.json
 * -Dkeycloak.import=/tmp/realm1.json,/tmp/realm2.json
 
+NOTE: The mechanism based on `keycloak.import` cannot be used together with the other mechanism based on `keycloak.migration.X` properties.
+If it is used together, the `keycloak.import` will be ignored. The `keycloak.import` mechanism always ignore the realms, which already
+exists in {project_name}. Generally the `keycloak.import` mechanism can be convenient for some development purposes, but if more
+flexibility is needed, it is recommended to stick with the mechanism based on `keycloak.migration.X` properties.
```

Comment:
Change "it is recommended to" to "we recommend that you"

Reply:
I've addressed this.